tox.ini

```diff
@@ -39,31 +39,13 @@ envlist=
 
 [testenv]
 deps =
-    ; Some relatively standard dependencies that we want to have
-    ; We cannot use extras here, because it is not supported on Py3
-    {alldeps,tests,coverage}: cryptography >= 0.9
-    {alldeps,tests,coverage}: pyopenssl >= 16.0.0
-    {alldeps,tests,coverage}: service_identity
-    {alldeps,tests,coverage}: idna >= 0.6
-    {alldeps,tests,coverage}: pyserial
-    {alldeps,tests,coverage}: python-subunit
-    {alldeps,tests,coverage}: pycrypto
-    {alldeps,tests,coverage}: appdirs
-    {alldeps,tests,coverage}: h2
-    {alldeps,tests,coverage}: priority
-    {alldeps,tests,coverage}: sphinx
+    {alldeps,tests,coverage}: twisted[all_non_platform]
 
     py27-{alldeps,tests,coverage}-{posix,macos}: pysqlite
-    py27-{alldeps,tests,coverage}: soappy
 
-    windows: pypiwin32
+    windows: twisted[windows_platform]
```

Comment:
The more I think about this, the more I think that this should be `.[windows_platform]`. Verifying this should be pretty simple.